docs-src/go/how-to-describe-a-schedule-in-go.md

```diff
@@ -0,0 +1,22 @@
+---
+id: how-to-describe-a-schedule-in-go
+title: How to describe a Schedule in Go
+sidebar_label: Describe Schedule
+description: Describe a Schedule in Go.
+---
+
+`Describe` retrieves information about the current Schedule configuration.
+This can include details about the Schedule Spec (such as Intervals), CronExpressions, and Schedule State.
```

Comment:
I can put it in the parentheses, then

Reply:
```suggestion
This can include details about the Schedule Spec (such as Intervals or, CronExpressions)
```suggestion
This can include details about the Schedule Spec (such as Intervals), CronExpressions, and Schedule State.
``` and Schedule State.
```